styleguide/backstop.json

```diff
@@ -174,18 +174,6 @@
       "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/patterns/02-molecules-article-stub-article-stub-for-category-page-with-image/02-molecules-article-stub-article-stub-for-category-page-with-image.html",
       "selectors": ".ama__article-stub"
     },
-    {
-      "label": "article stub for homepage",
-      "url": "http://localhost:3000/patterns/02-molecules-article-stub-article-stub-for-homepage/02-molecules-article-stub-article-stub-for-homepage.html",
-      "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/patterns/02-molecules-article-stub-article-stub-for-homepage/02-molecules-article-stub-article-stub-for-homepage.html",
-      "selectors": ".ama__article-stub"
-    },
-    {
-      "label": "article stub as hero for homepage",
-      "url": "http://localhost:3000/patterns/02-molecules-article-stub-article-stub-as-hero-for-homepage/02-molecules-article-stub-article-stub-as-hero-for-homepage.html",
-      "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/patterns/02-molecules-article-stub-article-stub-as-hero-for-homepage/02-molecules-article-stub-article-stub-as-hero-for-homepage.html",
-      "selectors": ".ama__article-stub"
-    },
```

Comment:
This has been removed by accident it seems.

Reply:
yep. restoring now.